fiat-ldap/src/main/java/com/netflix/spinnaker/fiat/config/LdapConfig.java

```diff
@@ -57,5 +57,9 @@ SpringSecurityLdapTemplate springSecurityLdapTemplate() throws Exception {
     String userSearchFilter;
     String groupSearchFilter = "(uniqueMember={0})";
     String groupRoleAttributes = "cn";
+
+    int thresholdToUseGroupFilter = 1000;
+    String groupsFilter = "";
+    String groupsUserAttributes = "member";
```

Comment:
Nit: Standardise the name with `groupsRoleAttributes`, something like `groupUserAttributes`?

Reply:
Also put this below `groupRoleAttributes`